### DISCUSSION TITLE
Is the stress api in Zepp OS 1.0 still available?

### DISCUSSION THREAD

**Discussion by JohnnyGo-x** [zepp-health/zeppos-forum#104] [Q&A]
The stress api in ZeppOS 1.0 document still shows "STRESS 传感器暂不可用，预计下一个 OTA 版本修复" in Chinese version, but English version document is not affected.

So, Is the stress api available?

**Comment by MyoungXUE** [comment]
Hello, the Stress sensor is still not available on Zepp OS 1.0 devices, we have fixed this issue on Zepp OS 2.0 supported devices. I will mark the 1.0 document as unavailable.

**Reply by JohnnyGo-x** [reply]
So stress level can't be reached in both app and watchface now? If this is a bug, will stress api be fixed in Zepp OS 1.0 sometime later?
